core-aam/core-aam.html

```diff
@@ -734,10 +734,10 @@
                         <td>
 				<code>ROLE_SYSTEM_GROUPING</code> + object attribute <code>xml-roles:feed</code></td>
                         <td>
-                            Need to verify. 
-			    <ul><li>Control Type: <code>Group</code></li>
-                                <li>Localized Control Type: <code>"feed"</code></li>
-                            </ul>
+			                    <ul>
+                            <li>Control Type = <code>Group</code></li>
+                            <li>Localized Control Type = "<code>feed</code>"</li>
```

Comment:
There is no reason not to match existing style guidelines, my fault - didn't know they exist and current UIA mappings were mixed. I'll definitely use "is" from now on